source/views/sis/course-search/detail/index.js

```diff
@@ -82,28 +97,38 @@ function Information({course}: {course: CourseType}) {
 }
 
 function Schedule({course}: {course: CourseType}) {
-	if (!course.times) {
+	if (!course.offerings) {
 		return null
 	}
-	const times = convertTimeStringsToOfferings(
-		{
-			times: course.times,
-			locations: course.locations,
-		},
-		{groupBy: 'day'},
-	)
-	const schedule = times.map(time => {
-		const hours = time.times.map(time => {
-			const start = moment.tz(time.start, 'hmm', CENTRAL_TZ).format('h:mm A')
-			const end = moment.tz(time.end, 'hmm', CENTRAL_TZ).format('h:mm A')
+	const groupedByDay = groupBy(course.offerings, 'day')
+	const schedule = map(groupedByDay, (offerings, day) => {
+		const timesFormatted = offerings.map(offering => {
+			const start = moment
+				.tz(offering.start, 'H:mm', CENTRAL_TZ)
+				.format('h:mm A')
+			const end = moment.tz(offering.end, 'H:mm', CENTRAL_TZ).format('h:mm A')
 			return `${start} – ${end}`
 		})
+		const locations = offerings.map(offering => offering.location)
+
+		const timelocs = zip(timesFormatted, locations)
+
+		const timelocsObj = timelocs.map(timelocs =>
+			zipObject(['time', 'location'], timelocs),
```

Comment:
Same thing about string vs function arguments here - `[o => o.time, o => o.location]`

Reply:
AFAIK the first argument to the `zipObject` function is just supposed to be an array of the property names that you want for the items of the arrays that you are zipping. So in this case the string arguments would be correct?

Reply:
@hawkrives

Reply:
It accepts functions, too (everything that accepts string shorthands also accepts functions, since lodash predates arrow functions and really needed the shorthand versions back then)

Reply:
But what is the object being passed to each function argument then? `[o => o.time, o => o.location]` doesn't work and `timelocs` is an array of arrays... @hawkrives

Reply:
Oh! I'm sorry. Completely misread what this does. 

Okay, new suggestion: 

```
timelocs.map(([time, location]) => ({time, location}))
```

I think it does the same thing as your zipObjects call? 
https://runkit.com/hawkrives/5b7e1387bd33fc00120840c2

Reply:
yeah that works!